argilla-server/src/argilla_server/bulk/records_bulk.py

```diff
@@ -189,7 +188,7 @@ async def upsert_records_bulk(
                     record.fields = jsonable_encoder(record_upsert.fields)
 
                 if self._db.is_modified(record):
-                    record.updated_at = datetime.now(UTC)
+                    record.updated_at = datetime.utcnow()
```

Comment:
I believe they will deprecate `datetime.utcnow()` and use `datetime.now(UTC)` instead. Perhaps we can already change it or do you want to tackle this in another PR across the entire repo?

https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated